clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -219,6 +219,44 @@ static Attr *handleSYCLIntelFPGADisableLoopPipeliningAttr(Sema &S, Stmt *,
   return new (S.Context) SYCLIntelFPGADisableLoopPipeliningAttr(S.Context, A);
 }
 
+// Handle [[intel:fpga_pipeline]] attribute.
+bool Sema::checkPipelineAttrArgument(Expr *E,
+                                     const SYCLIntelFpgaPipelineAttr *TmpAttr,
```

Comment:
Parameter is not used within the function.

Reply:
This function is redundant. removed